widgetssdk/src/main/java/com/glia/widgets/helper/Utils.java

```diff
@@ -508,6 +566,13 @@ private static <T> T getConfiguration(
         return newConf != null ? newConf : oldConf;
     }
 
+    public static UiTheme getFullHybridTheme(RemoteConfiguration remoteConfiguration, UiTheme newTheme, UiTheme oldTheme) {
+        UiTheme defaultUiTheme = UiTheme.getDefaultUiTheme();
+        UiTheme fullHybridOldTheme = getFullHybridTheme(oldTheme, defaultUiTheme);
+        UiTheme theme = getFullHybridTheme(newTheme, fullHybridOldTheme);
+        return ThemeUnits.apply(theme, remoteConfiguration);
+    }
```

Comment:
It's a bit confusing to me that `getFullHybridTheme()` is called twice. And I think there should be better names instead of `newTheme` and `oldTheme`.